internal/enums/event_outcome.go

```diff
@@ -0,0 +1,10 @@
+package enums
+
+// ENUM(
+// Unknown,
+// Success,
+// Failure,
+// Valid,
+// Invalid,
+// ).
+type EventOutcome string
```

Comment:
I feel like `Success` and `Valid` overlap with each other and that `Unknown` is not used. I prefer something similar to the workflow outcome values, for example: `Success`, `ValidationFailure` and `SystemFailure`.

Reply:
The "Unknown" value was inspired by Enduro, which often has a first enum value of "Unspecified" (e.g. https://github.com/artefactual-sdps/enduro/blob/main/internal/enums/preservation_task.go). I chose "Unknown" based on a faulty memory of the Enduro `enums.PreservationTask` value, I think "Unspecified" is better.  

I think the default value should not assume an actual outcome like "Success" or "Failure", because when an event instance is created the event is not complete, so it has no outcome.  I will change the Event constructor to set the default value of `Event.Outcome` to `Unspecified`.

I'll change the other values to Success, ValidationFailure and SystemFailure, as you've suggested.